migration-guide.md

```diff
@@ -42,25 +60,114 @@ For example, consider the following 1.4 request:
 }
 ```
 
-This report will query ten campaign items. For each of those, it will query one hundred geocity items. Finally, for each city, it will query the top 100 products. This report has the potential to return 10,000,000 records. In addition, it will try to get five metrics for each of those items! Finally, it will try to do that for each day going back a few years.
+This 1.4 report will query ten campaign items. For each of those, it will query one hundred geocity items. Finally, for each city, it will query the top 100 products. This report has the potential to return 10,000,000 records. In addition, it will try to get five metrics for each of those items. Finally, it will try to do that for each day going back a few years.
+
+### Example 2.0 request 
 
-When making requests to the `/reports` endpoint, smaller requests are better. The above example could be reduced into the following requests that can be made in parallel:
+Requests to the 2.0 `/reports` endpoint should be smaller and be made in parallel:
 
 1. Request the top ten `trackingCode` values for the given time period.
-2. For each of the values returned, request a breakdown of the top 100 `geocity` values
-3. For each of those values, request a breakdown of the top 100 `page` values
-4. Finally, a separate report for each metric. This means you go through steps 1-4 for `pageviews`, then again for `visits`, and so on.
+2. For each of the values returned, request a breakdown of the top 100 `geocity` values.
+3. For each of those values, request a breakdown of the top 100 `page` values.
+4. Request a separate report for each metric. This means you go through steps 1-4 for `pageviews`, then again for `visits`, and so on.
+
+#### First request
+
+```json
+{
```

Comment:
The following 4 examples are all using the 1.4 report format. These need to be converted to use the 2.0 version.

Reply:
okay. thanks. I will remove them.

Reply:
The following guide has examples of doing multiple breakdowns with the 2.0 reporting API format: https://github.com/AdobeDocs/analytics-2.0-apis/blob/master/reporting-multiple-breakdowns.md